doc/src/modules/integrals/integrals.rst

```diff
@@ -46,12 +52,34 @@ Exponential-polynomial functions. These multiplicative combinations of polynomia
 
 even a few nonelementary integrals (in particular, some integrals involving the error function) can be evaluated::
 
-	>>> integrate(exp(-x**2)*erf(x), x)
+	 >>> integrate(exp(-x**2)*erf(x), x)
 	  ____    2
 	\/ pi *erf (x)
 	--------------
 	      4
 
+SymPy's numeric integration (from mpmath) is very accurate and useful and would be preferred
+by many users over using lambdify/scipy. Here is how it is used:
+    >>> from sympy import *
+    >>> x=Symbol('x')
+    >>> integral = integrate(cos(x),(x,0,1))
+    >>> integral
+    sin(1) 
+    >>> integral.evalf()
+    0.841470984807897 
```

Comment:
This is still computing the integral symbolically. That is precisely not what is meant by https://github.com/sympy/sympy/issues/23025